composio/client/__init__.py

```diff
@@ -616,6 +618,18 @@ def get(  # type: ignore
         actions = actions or []
         apps = apps or []
         tags = tags or []
+        print("Apps - ", apps)
```

Comment:
Should we be having print statements? Or do we don't have a logger in SDK?

Reply:
removed